sportnet/sportnet/playgrounds/tests/test_views.py

```diff
@@ -1,10 +1,74 @@
+from django.core.management import call_command
 from django.test import TestCase
 from django.urls import reverse
 
+from datetime import datetime
+
+from sportnet.playgrounds.models import Event, Sport
+from sportnet.geoapp.models import City, Region
+from sportnet.authapp.tests.factories import UserFactory, user_passwords
+
 
 class IndexTest(TestCase):
 
     def test_page_code(self):
         url = reverse('playgrounds:index')
         response = self.client.get(url)
         self.assertEqual(response.status_code, 200)
+
+
+class EventReadDeleteControllerTest(TestCase):
+    def setUp(self):
+        call_command('flush', '--noinput')
+        self.user = UserFactory()
+        self.user_1 = UserFactory()
+        self.password = user_passwords[0]
+        region = Region.objects.create(title='test region')
+        City.objects.create(title='test city', region=region)
+        self.time = datetime(2019, 7, 11, 22, 0)
+        self.sport = Sport.objects.create(title='football',
+                                          description='sport desc')
+        self.event = Event.objects.create(title='test title', city_id=1,
+                                          sport=self.sport, time=self.time,
+                                          description='test description',
+                                          cost=100)
+        self.event.admins.add(self.user)
+
+    def test_display_read_event(self):
+        response = self.client.get(reverse('playgrounds:read',
+                                           kwargs={'pk': self.event.pk}))
+        self.assertContains(response, 'test title')
+
+    def test_delete_by_no_admin(self):
+        self.client.login(username=self.user_1.email,
+                          password=user_passwords[0])
+        response = self.client.get(reverse('playgrounds:delete',
+                                           kwargs={'pk': self.event.pk}))
+        self.assertEqual(response.status_code, 403)
+        post_response = self.client.post(reverse('playgrounds:delete',
+                                                 kwargs={'pk': self.event.pk}))
+        self.assertEqual(post_response.status_code, 403)
```

Comment:
```suggestion
        self.assertEqual(post_response.status_code, 403)
        self.assertTrue(Event.objects.filter(pk=self.event.pk).exists())
```
Необходимо проверить, что удаления не произошло. Сейчас проверяется только 403 код. Но может быть ситуация когда пользователь получит 403 код, но само удаление в базе произойдет. (если в ручную писать контроллер и сделать ошибку в структуре)